lib/muffin_man/fulfillment_outbound/v20200701.rb

```diff
@@ -0,0 +1,57 @@
+module MuffinMan
+  module FulfillmentOutbound
+    class V20200701 < SpApiClient
+
+      # To get outbound fulfillment preview
+      # @param [MuffinMan::RequestHelpers::OutboundFulFillment::FulfillmentPreviewRequest] fulfillment_preview_request in the form of object
+      def get_fulfillment_preview(fulfillment_preview_request)
+        @local_var_path = "/fba/outbound/2020-07-01/fulfillmentOrders/preview"
+        @request_body = fulfillment_preview_request.to_camelize
+        @request_type = "POST"
+        call_api
+      end
+
+      # To create outbound fulfillment order
+      # @param [MuffinMan::RequestHelpers::OutboundFulFillment::FulfillmentOrderRequest] fulfillment_order_request in the form of object
+      def create_fulfillment_orders(fulfillment_order_request)
+        @local_var_path = "/fba/outbound/2020-07-01/fulfillmentOrders"
+        @request_body = fulfillment_order_request.to_camelize
+        @request_type = "POST"
+        call_api
+      end
+
+      # To get list of fulfillment orders
+      # @param [String] query_start_date optional
+      # @param [String] next_token optional
+      def list_fulfillment_orders(query_start_date=nil,next_token=nil)
```

Comment:
```suggestion
      def list_all_fulfillment_orders(query_start_date: nil, next_token: nil)
```
Same thing here. 

Also, can we make these optional params symbolized? That way if you are passing `next_token` you don't have to do `list_fulfillment_orders(nil, "next_token")` but can instead just do `list_fulfillment_orders(next_token: "next_token")`. It's not ideal to have to pass in nil for an optional param.

Reply:
Added the changes